### PR TITLE
Basic Issue Template with Discord Link

### PR DESCRIPTION
**Pull request by vulcandth**
Please reference: https://github.com/pret/pokecrystal/issues/930

To see what this looks like; check out the example repo I set up: https://github.com/vulcandth/Pret-Test-Issue-Templates/issues/new/choose

Edit: Above link updated to reflect latest commit.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/blank_issue.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+name: Blank Issue
+about: Create an issue with a blank template.
+---
```
Comment (Rangi42) [inline .github/ISSUE_TEMPLATE/blank_issue.md:4]: The goal is to make it clear to people that GitHub issues are for pokecrystal, not their own use of it, so:
```suggestion
---
name: Update to pokecrystal
about: Suggest a possible change to pokecrystal itself.
---
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: pret Discord
+    url: https://discord.gg/d5dubZ3
+    about: If you need Build Help, Romhack Help, or have General Repo Questions, ask in our discord instead!
```
Comment (Rangi42) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: If you need help with using pokecrystal, or just have general questions about it, ask in our Discord server!
```

**Comment by Rangi42** [comment]
Looks good to me! @mid-kid Is this a good template we could add to the other repos too?

**Comment by dannye** [comment]
Yep, lgtm.

**Comment by mid-kid** [comment]
~~Final plea for a "Can't use Discord? Try your luck on IRC"~~

lgtm

**Pull request merged**